aten/src/ATen/quantized/TensorFactories.cpp

```diff
@@ -0,0 +1,15 @@
+#include <ATen/ATen.h>
+#include <c10/core/TensorOptions.h>
+#include <c10/core/QScheme.h>
+#include <ATen/quantized/Quantizer.h>
+
+namespace at {
+namespace native {
+
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ empty ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// TODO: Tensor empty_quantized_cpu(IntArrayRef size, const TensorOptions& options, QScheme qscheme) {
+Tensor empty_affine_quantized_cpu(IntArrayRef size, const TensorOptions& options, double scale, int64_t zero_point) {
```

Comment:
the reason this function exists is that we don't yet have bindings of quantizer to python, right? it might be more natural to eventually just create quantizer object and pass it to generic `empty_quantized` function. But I'm not sure it's worth pushing from atm.

Reply:
we discussed with Michael and he feels it's unnecessary and also exposing quantizer in python frontend is too complicated since we need some polymorphic python object right? We can revisit later if there is a need.

Reply:
For now we'll just use this until some binding features are supported by @suo

Reply:
I'd add a comment/todo about it as it's not obvious why we don't just take quantizer here.